src/router/index.js

```diff
@@ -2,12 +2,16 @@ import Vue from 'vue'
 import Router from 'vue-router'
 import Login from '@/components/Login'
 import ProjectList from '@/components/ProjectList'
+import ActorList from '@/components/ActorList'
```

Comment:
If possible, I'd suggest keeping this list of imports in order

Reply:
@taliaga Organised the imports a bit better, though route definitions have to be in order of priority to avoid missmatches (project/1/services/new VS project/1/services/1)